bootstrap.cmd

```diff
@@ -3,6 +3,7 @@
 where /q dotnet.exe || echo Can't find dotnet.exe! Please add to PATH && goto :EOF
 where /q git.exe || echo Can't find git.exe! Please add to PATH && goto :EOF
 set root=%~dp0
+pushd %root%
```

Comment:
It was for how I was planning on running this in the lab. I don't know if it's necessarily correct, but I wanted to be able to call bootstrap as a peer of coreclr (making jitutils a peer), but call it without having to change directories in the lab job. I just didn't like the idea of having jitutils as a subdirectory of coreclr in the lab job.